### DISCUSSION TITLE
🚀 Feature Request: Regexp support in D1

### DISCUSSION THREAD

**Discussion by codehz** [cloudflare/workers-sdk#13545] [Feature Requests]
### Describe the solution

Currently, using regexp to match strings in D1 will get an error because the regexp function is not defined (by default)

> The REGEXP operator is a special syntax for the regexp() user function. No regexp() user function is defined by default and so use of the REGEXP operator will normally result in an error message. If an [application-defined SQL function](https://www.sqlite.org/appfunc.html) named "regexp" is added at run-time, then the "X REGEXP Y" operator will be implemented as a call to "regexp(Y,X)".

It would be very helpful to add the regex support to D1

**Comment by penalosa** [comment]
@rozenmd is this still on the roadmap?
